Source/WebCore/PAL/pal/crypto/openssl/CryptoDigestOpenSSL.cpp

```diff
@@ -113,6 +113,7 @@ CryptoDigest::~CryptoDigest()
 
 static std::unique_ptr<CryptoDigestContext> createCryptoDigest(CryptoDigest::Algorithm algorithm)
 {
+    std::unique_ptr<CryptoDigest> digest(new CryptoDigest);
```

Comment:
Can we use makeUnique instead of new?
Also, this seems unused.

Reply:
yea this is rebase remnant!! removed.